musicbrainz/getArtists.js

```diff
@@ -138,7 +138,7 @@ module.exports = {
             }
 
 
-        }, 3000)
+        }, 1200)
```

Comment:
Yep, and setting it to 1000 would therefore result in a couple of blocked requests